msbuild/Xamarin.MacDev.Tasks.Core/Tasks/DittoTaskBase.cs

```diff
@@ -44,6 +45,20 @@ protected override string GenerateCommandLineCommands ()
 			return args.ToString ();
 		}
 
+		public override bool Execute ()
+		{
+			if (!base.Execute ())
+				return false;
+
+			if (TouchDestinationFiles) {
+				foreach (var file in Directory.EnumerateFiles (Destination!.ItemSpec, "*", SearchOption.AllDirectories)) {
```

Comment:
@rolfbjarne This won't touch files on Windows because it'll be executed remotely, but I'm not 100% if the files you're trying to update here exist on Windows though. If those are part of a MSBuild input/output check most probably they do exist on Windows.

Reply:
The "is file up-to-date" check that runs into this scenario is in our own code:

https://github.com/xamarin/xamarin-macios/blob/a9c2caa5710a76783ac82c0a589020477e7bc9c9/msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CodesignTaskBase.cs#L87-L101

So I don't think we need to replicate this on Windows.

Reply:
Oh, awesome! Yep, in that case we don't need this to happen on Windows :)